profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_dgt_connector/README.md

```diff
@@ -1,71 +1,70 @@
+supported by the original TMGMT module. This includes:
+* Nodes
+* Blocks
+* Beans
+* Views
+* Locale strings
+* Field instances
+* Files
+* Menu and menu links
+* Vocabularies and Taxonomy terms
 
-It is now possible to bundle other different translatable items
-in the ```Small Jobs Cart```, using the translator ```TMGMT DGT Translator```:
+The module supports strings of all sizes but Nodes whose length is bigger
+than 300 characters have to be sent on individual requests.
 
-* Content with length under 300 characters;
-* Menu and Menu items;
-* Taxonomies and Terms;
-* Blocks 
+Other strings can and should be bundled together into single requests.
+Even though any string can be bundled together, for consistency sake it is recommended
+that related strings be bundled together (all items of a single menu for example).
 
-To add one translatable item to the ``Small Jobs Cart``, 
-go to the translation page of that item,
-chose the target languages and then use the "Send to cart" button.
 
-Then, in the ```Small Jobs Cart (admin/tmgmt/dgt_cart)```, you can send a bundle to be translated.
-Each bundle is a set of items having the same target languages set.
-Before sending a bundle for translation, please edit the bundle, then revise and complete
-the context information: Context URL should be filled and also any other
-hints that may help the person in charge of translation to understand the
-context where the item will be displayed.
+# Usage
 
+Requesting translations for content whose length is bigger than 300 follows the usual workflow
+of the TMGMT module. Please refer to the documentation of the module for more details.
+
+In order to bundle items together a user must use the custom DGT Cart which replaces the
+default TMGMT Cart.
+A user can add a translatable string in two different ways: through the translation form
+of the content or through the Sources page provided by TMGMT.
+
+#### Translation forms
+Translation forms for most content and string types 
+can be found when editing the content itself under the tab "Translate" (for example,
+node translation pages can be found at node/NODE_ID/translate).
+Selecting the "Send to cart" option will add them to the cart for later usage.
+
+#### Sources page
+All translatable strings are available on the Sources page provided by the TMGMT module.
+The page can be found at admin/tmgmt/sources and it provides separate tabs for each
+string type.
+After selecting the desired strings, they can be sent to the cart itself by selecting the target
+languages and clicking the "Send to cart" button.
+
+#### DGT Cart
+
+The custom DGT cart can be accessed through the following url "admin/tmgmt/dgt_cart".
+The cart shows all the currently created bundles.
+Each bundle is created based on the target languages of the items sent into the cart.
+Once in a bundle, the items inside them can be edited and extra metadata added by using
```

Comment:
"them" is wrong because it is a single bundle.
I suggest:
"The Edit option allows to see the items inside a bundle, edit its context data and also to remove items.